lib/charms/smtp_integrator/v0/smtp.py

```diff
@@ -68,7 +68,8 @@ def __init__(self, *args):
 
 logger = logging.getLogger(__name__)
 
```

Comment:
Most libraries provide some docstrings with usage examples and small documentation, I think we should add some here

Reply:
I've expanded the existing one a bit. Let me know what you think